src/main/java/seedu/address/logic/commands/queue/AddConsultationRoomCommand.java

```diff
@@ -19,10 +19,10 @@
     public static final String COMMAND_WORD = "onduty";
     public static final String MESSAGE_INVALID_DOCTOR_ID = "The reference ID '%1$s' does not belong "
             + "to any registered doctor!";
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a new room. "
-            + "Parameters: "
-            + "REFERENCE_ID \n"
-            + "Example: " + COMMAND_WORD + " 001A";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Marks a doctor as on-duty "
+            + "and adds it to the list of displayed rooms.\n"
```

Comment:
Please rephrase this. There should be no reference to any sort of room. It only adds a list of staff doctors on duty.

Reply:
Rephrased